.env-devel

```diff
@@ -35,7 +35,7 @@ POSTGRES_PASSWORD=adminadmin
 POSTGRES_PORT=5432
 POSTGRES_USER=scu
 
-RABBIT_CHANNELS={"log": "comp.backend.channels.log", "progress": "comp.backend.channels.progress", "instrumentation": "comp.backend.channels.instrumentation"}
+RABBIT_CHANNELS={"log": "simcore.services.logs", "progress": "simcore.services.progress", "instrumentation": "simcore.services.instrumentation", "events": "simcore.services.events"}
```

Comment:
is it really necessary to change them all? or is for naming consistency?  If so, did you make sure that this change does not break the other channels?
I understand that you only need the "events" channel, right?

Reply:
With @sanderegg we agreed about this name change. Also this will trigger devops actions